pkg/cmd/listen.go

```diff
@@ -30,6 +30,8 @@ type listenCmd struct {
 	cmd       *cobra.Command
 	wsBaseURL string
 	noWSS     bool
+	ci        bool
+	apiKey    string
```

Comment:
Not sure why the apiKey / ci are still relevant here. The remnant from the previous implementation?

Reply:
Yup, the remnant of the previous implementation. My bad, I should have reviewed my own PR first and catch these.